index.html

```diff
@@ -40,15 +41,15 @@
 
         <div id="mobile-menu-container">
           <span class="close-icon" onclick="menuFunction()"
-            ><img src="assets/images/close-icon.svg" alt=""
+            ><img src="image/close-icon.svg" alt="closeicon"
           /></span>
           <ul id="mylink">
             <li>
               <a
                 href="javascript:void(0);"
                 onclick="menuOptions('#my-recent-work')"
-                >Projects</a
-              >
+                >Portfolio</a>
+              
             </li>
```

Comment:
- [ ] Good job implementing the mobile menu 😄, but as the previous reviewer said If you check your `font style` and Figma design `font style` it does not match so kindly fix this issue. 👍 
> It has to be something like `font-weight: 600;`

- [ ] Also,  If you check the spacing between the text it does not match to the Figma design so please fix this issue. 👍

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![mobilefig](https://user-images.githubusercontent.com/57408419/224742266-8921e687-e7f9-42a1-aeaf-b73e552871b4.jpg)| |![mobile](https://user-images.githubusercontent.com/57408419/224742161-bf3ad578-0739-47d2-a791-40acd6033231.jpg)| |